solr/core/src/java/org/apache/solr/response/transform/DocTransformers.java

```diff
@@ -88,4 +89,11 @@ public boolean needsSolrIndexSearcher() {
     return false;
   }
 
+  @Override
+  public void finish() {
+    for( DocTransformer a : children ) {
+      a.finish();
```

Comment:
yes? I think that if a transformer throws an exception solr request should fail (you can still try and catch the exception inside the transformer if you want to obtain 'silent' behaviour) what do you think?